src/util/fileSizeUtil.ts

```diff
@@ -0,0 +1,11 @@
+const measurements = ['B', 'KB', 'MB', 'GB', 'TB'];
```

Comment:
Jeg gjetter at det finnes et bibliotek som gjør dette.

Reply:
Det gjør det absolutt! F-eks `pretty-bytes`. Tenkte bare at det ikke var nødvendig å dra inn et ekstra bibliotek for dette, men kan godt bytte over til det ved behov.